src/Settings.php

```diff
@@ -1117,7 +1117,8 @@ public function settingsFieldInputBoolean($args)
             // Check if the value is true or false
             $checked = ($value == "1") ? "checked" : "";
 
-            // echo a proper input type="text"
+            // echo a proper input type="checkbox"
+            echo "<input type='hidden' name='".$settings_key."[$field_key]' value='0' autocomplete='off'/>";
```

Comment:
why are we having 2 inputs with same name here ?

Reply:
Yes, this is intentional to force saving the default value to 0 in the DB for the bool input.